input/fsh/profiles/CHEMEDCompositionMedicationTreatmentPlan.fsh

```diff
@@ -11,7 +11,11 @@ Description: "Definition of the composition for the medication treatment plan do
 * identifier.value ^example.label = "CH EMED"
 * identifier.value ^example.valueString = "urn:uuid:daa8cd41-34a1-4a9c-9a6d-cd3f850142e9"
 * status = #final (exactly)
-* type = $loinc#77603-9
+
+* type = $sct#761931002
+* type ^short = "761931002 Medication treatment plan report (record artifact)"
```

Comment:
Nitpicking: the other shorts don't have the code, only the display name :)

Reply:
Ou, thanks for that!